src/components/Users/UsersList.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useUsersContext } from '../../hooks/useUsersContext';
+import { User } from '../../types';
+
+type Props = {
+  onCloseDropdown: () => void;
+};
+
+export const UsersList: FC<Props> = ({ onCloseDropdown }) => {
+  const { onSelectUser, selectedUser, users } = useUsersContext();
+
+  const handleSelectUser = (user: User) => {
+    onSelectUser(user);
+    onCloseDropdown();
+  };
+
+  return (
+    <>
+      {users.map((user) => (
+        <a
+          href={`#user-${user.id}`}
+          className={classNames('dropdown-item', {
+            'is-active': selectedUser?.id === user.id,
+          })}
+          key={user.id}
+          onClick={() => handleSelectUser(user)}
```

Comment:
It's better to prevent default browser action for onClick event in the <a> tag, to avoid unnecessary page jumps or reloads.